Initium-Odp/src/com/universeprojects/miniup/server/commands/CommandSetLeader.java

```diff
@@ -45,7 +45,7 @@ public void run(Map<String, String> parameters) throws UserErrorMessage
 		Long charId;
 		try {
 			charId = Long.parseLong(parameters.get("charId"));
-		} catch (Exception e) {
```

Comment:
I would catch any exception here like you had before since any exception would always mean a malformed charId anyway.